freebsd_sysctl/__init__.py

```diff
@@ -95,14 +95,16 @@ def size(self) -> int:
         return self._size
 
     @property
-    def raw_value(self) -> int:
+    def raw_value(self) -> typing.Any:
```

Comment:
This could be `typing.Union[str, int]` 🤔

Reply:
By the way, I'm interested in supporting struct types.

Dict looks a good return type for most of structs but there are few exceptions like avgload where it returns a list of float.

If you don't plan supporting structs, it will be nicer if value() can return bytes so that user side can do a conversion, etc.

Reply:
> This could be `typing.Union[str, int]` thinking

By the way, "def query_value() -> bytes" but calls ctl_type() at end and ctl_type() returns CtlType type.

I wasn't sure what type is exactly coming.
It wasn't definitely not int type and placed typing.Any as a starter.

Reply:
Let's address the invalid typing in another PR.

> By the way, I'm interested in supporting struct types.

Indeed, me too. Would be inviting for unit tests (once they are fixed 🙃 ).